src/views/Card/Card.js

```diff
@@ -66,7 +66,13 @@ export default class Card extends Component {
     /** Shorthand for CardMeta. */
     meta: customPropTypes.itemShorthand,
 
-    /** Render as an `a` tag instead of a `div` and called with event on Card click. */
+    /**
+     * Called after user's click. When passed, the component render as an `a`
+     * tag by default instead of a `div`.
+     *
+     * @param {SyntheticEvent} event - React's original SyntheticEvent.
+     * @param {object} data - Relevant data.
```

Comment:
I used the phrase "Relevant data" as an example to convey the intent in the spec, however, I think we should be as descriptive as possible in the actual docs.  In this case, we can say `data` is props since that is all that is being passed.